### PR TITLE
Nua key activity log page

### PR DESCRIPTION
**Pull request by mpellerin42**
- NUA Activity logs page
- Access account-nua from its own route
- Add activity button to understanding API key
- Use $gray-contrast instead of #707070

### CODE REVIEW

File: apps/app/src/app/account/account-nua/account-nua.component.ts [hunk 1]
```diff
@@ -16,57 +15,68 @@ import { TokenDialogComponent } from '../../components/token-dialog/token-dialog
...
+        switchMap(() => this.nua.renewClient(client.client_id)),
       )
       .subscribe(({ token }) => {
         this.showToken(token);
         this.nua.updateClients();
       });
   }
 
+  goToActivity(client: NUAClient): void {
+    const currentPath = location.pathname;
```
Comment (ebrehault) [inline apps/app/src/app/account/account-nua/account-nua.component.ts:55]: This approach is a bit hacky, we have a navigation service to provide proper route path. Go that way:
```ts
accountUrl = this.stateService.account.pipe(
    filter((account) => !!account),
    map((account) => this.navigation.getAccountUrl(account!.slug)))
```

File: apps/app/src/app/account/account-nua/nua-activity/nua-activity.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+export interface Activity {
+  file: string;
+  timestamp: string;
+  actor: string;
+}
+
+@Component({
+  selector: 'app-nua-activity',
+  templateUrl: './nua-activity.component.html',
+  styleUrls: ['./nua-activity.component.scss'],
```
Comment (ebrehault) [inline apps/app/src/app/account/account-nua/nua-activity/nua-activity.component.ts:18]: OnPush

**Review by ebrehault — CHANGES_REQUESTED** [review]

File: apps/app/src/app/account/account-nua/account-nua.component.html [hunk 3]
```diff
@@ -32,6 +32,12 @@
                              stfTooltip
                              stfTooltipText="account.generate_key">
           </app-button-action>
+          <app-button-action (click)="goToActivity(client)"
```
[the remark this answers is not in the document]
Comment (mpellerin42) [reply apps/app/src/app/account/account-nua/account-nua.component.html:34]: Good idea!

File: apps/app/src/app/account/account-nua/account-nua.component.ts [hunk 4]
```diff
@@ -56,8 +59,11 @@ export class AccountNUAComponent {
   }
 
   goToActivity(client: NUAClient): void {
-    this.stateService.account
+    this.isNuaActivityEnabled
```
Comment (ebrehault) [inline apps/app/src/app/account/account-nua/account-nua.component.ts:62]: a bit overkill (the button is invisible if the feature is not enabled), but no big deal

**Review by ebrehault — APPROVED** [review]

File: apps/app/src/app/account/account-nua/account-nua.component.ts (hunk 4 above)
Comment (mpellerin42) [reply apps/app/src/app/account/account-nua/account-nua.component.ts:62]: I know, it's a security habit I have from Onna 😅 
And actually, I didn't go through it all as I didn't protect the ability to route to nua activity page with a guard, but I felt like that was good enough 😉

**Review by operramon — APPROVED** [review]

**Pull request merged**